lib/taurus/qt/qtgui/table/taurusgrid.py

```diff
@@ -194,6 +194,18 @@ def get_readwrite_models(expressions, limit=1000):
     models = models[:limit]
     return models
 
+class TaurusGridCell(Qt.QFrame):
```

Comment:
Since this class seems to be a very private helper for `TaurusGrid`, I would explicitly mark it as private by calling it `_TaurusGridCell` (this way we avoid having to worry about API compatibility if we refactor in the future)